NitroxClient/GameLogic/InitialSync/CyclopsInitialAsyncProcessor.cs

```diff
@@ -24,14 +24,16 @@ public CyclopsInitialAsyncProcessor(Vehicles vehicles)
 
         public override IEnumerator Process(InitialPlayerSync packet, WaitScreen.ManualWaitItem waitScreenItem)
         {
-            this.waitScreenItem = waitScreenItem;
-            vehicles.VehicleCreated += OnVehicleCreated;
+            IEnumerable<VehicleModel> cyclops = packet.Vehicles.Where(v => v.TechType.Enum() == TechType.Cyclops);
```

Comment:
```suggestion
            List<VehicleModel> cyclopses = packet.Vehicles.Where(v => v.TechType.Enum() == TechType.Cyclops).ToList();
```

Should do `ToList` because an IEnumerable will be lazily evaluated when created from LINQ. Then you have acces to the faster `cyclopses.Count` too